### DISCUSSION TITLE
Compiler

### DISCUSSION THREAD

**Discussion by CorruptionHades** [TodePond/GulfOfMexico#173] [General]
I made a "compiler" using java, havent implemented all features yet. Dunno if anyone cares lol
https://github.com/CorruptionHades/DreamBeardCompiler

Features so far:
- variable creation (var var, const const, var const, const var)
- print function
- delete keyword
- creation of custom functions

**Comment by stohrendorf** [comment]
We also have #74 for this.

**Comment by TodePond** [comment]
Very nice! Good luck to you :)

**Comment by CorruptionHades** [comment]
should i remake it in c# and make it compile to an exe

**Reply by TodePond** [reply]
follow your heart
